blueprints/app/files/tests/helpers/start-app.js

```diff
@@ -5,8 +5,8 @@ import config from '../../config/environment';
 export default function startApp(attrs) {
   let application;
 
-  let attributes = Ember.merge({}, config.APP);
-  attributes = Ember.merge(attributes, attrs); // use defaults, but you can override;
+  let attributes = (Ember.assign || Ember.merge)({}, config.APP);
+  attributes = (Ember.assign || Ember.merge)(attributes, attrs); // use defaults, but you can override;
```

Comment:
should we do something like `let assign = Ember.assign || Ember.merge; let attributes = assign(...)` instead to reduce code duplication?